docs/README.md

```diff
@@ -33,20 +33,24 @@ Once installed, the app will generate a new route called `/stores` for your stor
 
 The new page already contains a default template with all blocks exported by the Store Locator app, meaning that the `/stores` page is ready to be rendered and no further actions are required. However, you can **customize the new page overwriting the template by creating a brand new one as you wish**. To do so, check the **Advanced configurations** section below.
 
+This app will also add a new entry to the `/sitemap.xml` file to make all your Stores accessible to the Search Engines.
+
+Make sure you have the `vtex.store-sitemap@2.x` App installed on your account to make use of the sitemap feature.
+
 ### Advanced configuration
 
 In order to define the Quickorder custom page UI, you must use the blocks exported by the `vtex.quickorder` app. Namely, they are:
 
-| Block name | Description |
-| :--------: | :---------: |
-| `store-list` | Renders a list of retail stores and a map with all their localization. | 
-| `store-group` |  Provides the pickup point data to other blocks exported by the app, such as the ones listed below. |
-| `store-name` | Renders the store name. | 
-| `store-back-link` |  Renders a link to return to the previous page. | 
-| `store-map` | Renders a map with the retail store localization. | 
-| `store-address` | Renders the store's address. | 
-| `store-hours` | Renders the store's opening hours. | 
-| `store-instructions` | Renders the desired instructions to access the retail store. | 
+|      Block name      |                                            Description                                             |
+| :------------------: | :------------------------------------------------------------------------------------------------: |
+|     `store-list`     |               Renders a list of retail stores and a map with all their localization.               |
+|    `store-group`     | Provides the pickup point data to other blocks exported by the app, such as the ones listed below. |
+|     `store-name`     |                                      Renders the store name.                                       |
+|  `store-back-link`   |                           Renders a link to return to the previous page.                           |
+|     `store-map`      |                         Renders a map with the retail store localization.                          |
+|   `store-address`    |                                    Renders the store's address.                                    |
+|    `store-hours`     |                                 Renders the store's opening hours.                                 |
+| `store-instructions` |                    Renders the desired instructions to access the retail store.                    |
```

Comment:
No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!